pkg/csplugin/watcher.go

```diff
@@ -0,0 +1,55 @@
+package csplugin
+
+import (
+	"time"
+
+	"github.com/crowdsecurity/crowdsec/pkg/models"
+)
+
+type PluginWatcher struct {
+	PluginConfigByName     map[string]PluginConfig
+	AlertCountByPluginName map[string]int
+	C                      chan string
```

Comment:
can we have a more explicit name for the chan ? this one might be confused with ticker

Reply:
fixed via https://github.com/crowdsecurity/crowdsec/pull/857/commits/bf954ccd25869f1a631a588177d18f3ccd14bc38